Cargo.toml

```diff
@@ -35,17 +34,15 @@ clear_on_drop = { version = "0.2", default-features = false }
 hex = "0.4"
 criterion = "0.3"
 bincode = "1"
-rand_chacha = "0.3"
+rand_chacha = "0.3.1"
 
 [features]
 default = ["std"]
-avx2_backend = ["curve25519-dalek/avx2_backend"]
```

Comment:
Because the new version of curve25519-dalek that this branch uses no longer has a `avx2_backend` feature flag (see [here](https://docs.rs/curve25519-dalek/4.0.0-rc.1/curve25519_dalek/#feature-flags)).